docs/manual/java/guide/build/index.toc

```diff
@@ -1,2 +1,8 @@
 LagomBuild:Defining a Lagom build
-MultipleBuilds:Splitting a system into multiple builds
\ No newline at end of file
+MultipleBuilds:Splitting a system into multiple builds
```

Comment:
Good idea! The index is a bit different now, but I've used next links for all of the integrating topics.